spec/vcr_cassettes/js_fetcher.yml

```diff
@@ -0,0 +1,2763 @@
+---
```

Comment:
This entire cassette can be removed as it isn't being used by the `js_fetcher` spec.  You may want to save it to a local branch in your fork if you want to reference it in the vcr config ticket you're creating/will create.  But, leaving it here is unnecessary and potentially confusing in the future.